### ISSUE TITLE
Abnormal circulating Peptidase D concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6003]
**Preferred term label:** Abnormal circulating Peptidase D concentration

**Synonyms** Abnormal circulating prolidase

**Definition (free text, please give PubMed ID)** Concentration of peptidase D in the blood is outside the normal range., PMID: 27546702

**Parent term (use hpo.jax.org/app)** Abnormality of circulating enzyme level HP:0011021

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Peptidase D concentration

**Synonyms** Decreased circulating prolidase

**Definition (free text, please give PubMed ID)** Concentration of peptidase D in the blood is below the normal range., PMID: 1943683

**Parent term (use hpo.jax.org/app)** Abnormal circulating Peptidase D concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Peptidase D concentration

**Synonyms** Increased circulating prolidase

**Definition (free text, please give PubMed ID)** Concentration of peptidase D in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Peptidase D concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is not a clinical test that has a normal range such that an individual can display an "abnormal" value for it. A good way of deciding whether a lab test is a valid HPO term is to check whether there is a LOINC code for the corresponding test. If there is, that means that it is being tested clinically in at least some situations.
